docs/en/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md

```diff
@@ -4,10 +4,27 @@ displayed_sidebar: "English"
 
 # RESUME ROUTINE LOAD
 
+## Description
+
+Resume a Routine load job. The job will temporarily enter **NEED_SCHEDULE** state because the job is being re-scheduled. And after some time, the job will be resumed to **RUNNING** state, continuing consuming messages from the data source and loading data. You can check the job's information with the [SHOW ROUTINE LOAD](https://docs.starrocks.io/docs/3.2/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD/) statement.
+
+## Syntax
+
+```SQL
+RESUME ROUTINE LOAD FOR [db_name.]<job_name>
+```
+
+## Parameters
+
+| **Parameter** | **Required** | **Description**                                              |
+| ------------- | ------------ | ------------------------------------------------------------ |
+| db_name       |              | The name of the database where the Routine Load job belongs. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/0dc1b3e1-4760-4bef-a43b-c45b8b362293)
-> to which